docs/user-guide/custom-themes.md

```diff
@@ -19,8 +19,10 @@ and their usage.
 
 ## Creating a custom theme
 
-The bare minimum required for a custom theme is a `main.html` [Jinja2 template]
-file which is placed in a directory that is *not* a child of the [docs_dir].
+The bare minimum required for a custom theme is a 
+[theme configuration file](#theme-configuration) 
+and a `main.html` [Jinja2 template]
+file, both placed in a directory that is *not* a child of the [docs_dir].
```

Comment:
The line length is 80 chars. I realize the long link messes things up a little, but this could look better. May I suggest using your editors tools to automatically hard wrap the lines.